sites/public/layouts/application.tsx

```diff
@@ -33,7 +34,7 @@ const Layout = (props) => {
         <SiteHeader
           skip={t("nav.skip")}
           logoSrc="/images/detroit-logo.png"
-          notice="This is a preview of our new website. We're just getting started. We'd love to get your feedback."
+          notice={<div dangerouslySetInnerHTML={getTranslationMarkup("nav.getFeedback")}></div>}
```

Comment:
Is this really dangerous?

Reply:
I don't believe so, based on [the documentation](https://reactjs.org/docs/dom-elements.html#dangerouslysetinnerhtml) and [this article](https://pragmaticwebsecurity.com/articles/spasecurity/react-xss-part2.html) I think that this is only dangerous if you're using unsafe (user generated) html.